subsys/bluetooth/host/smp.c

```diff
@@ -2020,7 +2020,9 @@ static void smp_timeout(struct k_work *work)
 	struct k_work_delayable *dwork = k_work_delayable_from_work(work);
 	struct bt_smp *smp = CONTAINER_OF(dwork, struct bt_smp, work);
 
-	LOG_ERR("SMP Timeout");
+	LOG_ERR("SMP Timeout (flags:0x%08x allowed_cmds:0x%08x)",
+		(unsigned int)atomic_get(&smp->flags[0]),
+		(unsigned int)atomic_get(&smp->allowed_cmds[0]));
```

Comment:
I would still suggest a comment as to why we only take `[0]` as well as a `BUILD_ASSERT` to ensure that we don't have more than `sizeof((unsigned int) * BITS_PER_BYTE)` bits in the array